cs-minibot-platform-src/src/main/java/simulator/SandboxOccupancyMatrix.java

```diff
@@ -0,0 +1,167 @@
+package simulator;
+import org.jbox2d.callbacks.QueryCallback;
+import org.jbox2d.collision.AABB;
+import org.jbox2d.collision.shapes.CircleShape;
+import org.jbox2d.collision.shapes.PolygonShape;
+import org.jbox2d.common.Vec2;
+import org.jbox2d.dynamics.*;
+import simulator.simbot.Dijkstras;
+import simulator.simbot.Dijkstras.Node;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+
+
+/**
+ * Created by CornellCup on 4/25/2017.
+ */
+public class SandboxOccupancyMatrix extends Thread{
+
+    public final static float occupancyMatrixBoxSize = 0.50f; //Size of box in meters
+    public static int occupancyMatrixWidth = 40; //Number of boxes
+    public static int occupancyMatrixHeight = 40; // Number of boxes
+
+    public static void main(String[] args) throws Exception{
```

Comment:
I'm assuming this is a testing file so I wont really review it, but maybe add a comment to make it clear it's for testing